pkg/apis/core/types.go

```diff
@@ -3761,6 +3761,27 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for the extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by kube-scheduler.
+type PodExtendedResourceClaimStatus struct {
+	// RequestMapping identifies the mapping of <container, extended resource backed by DRA> to  device request.
+	RequestMapping []ContainerExtendedResourceRequest
+
+	// ResourceClaimName is the name of the ResourceClaim that was
+	// generated for the Pod in the namespace of the Pod.
+	ResourceClaimName string
+}
+
+type ContainerExtendedResourceRequest struct {
+	// container name
```

Comment:
This comment doesn't add any information, let's at least be more verbose.
```suggestion
	// The name of the container requesting resources.
```